emails/views.py

```diff
@@ -84,15 +84,15 @@ def _index_POST(request):
             user=user_profile.user
         )
         if locked_profile.num_active_address >= settings.MAX_NUM_FREE_ALIASES:
-            if 'moz-extension' in request.headers.get('Origin', ''):
+            if not settings.SITE_ORIGIN in request.headers.get('Origin', ''):
```

Comment:
```suggestion
            if not (settings.SITE_ORIGIN in request.headers.get('Origin', '')):
```
Was what you were trying to check is that the request came from OUTSIDE of the Relay website? If so, my recommendation is to check like above.

Reply:
nvm, `in` is resolved before the `not`

```
>>> foo = ''
>>> bar = ['abc']
>>> if foo in bar:
...     print('foo in bar')
...
>>> if not foo in bar:
...     print('foo not in bar')
...
foo not in bar
>>> not foo
True
```